framework/auth/views.py

```diff
@@ -80,10 +80,10 @@ def forgot_password_post():
         if user_obj:
             #TODO: Remove this rate limiting and replace it with something that doesn't write to the User model
             now = datetime.datetime.utcnow()
-            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=FORGOT_PASSWORD_MINIMUM_TIME)
+            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=settings.FORGOT_PASSWORD_MINIMUM_TIME)
```

Comment:
Probably, but why ask for trouble? Saman already did this plus the other related changes. If someone messes up a merge conflict, it may not be immediately noticed, and the nature of this particular PR is one that a lot of little merge conflicts will build up that may be hard to distinguish.